packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -566,3 +859,57 @@ class ReusePipelineStack extends cdk.Stack {
     });
   }
 }
+
+const createPipeline = (scope: Construct, pipelineName?: string, pipelineId: string = 'Pipeline') => {
+  const artifact = new codepipeline.Artifact();
+  return new codepipeline.Pipeline(scope, pipelineId, {
+    pipelineName,
+    crossAccountKeys: true,
+    reuseCrossRegionSupportStacks: false,
```

Comment:
Pretty sure these are the defaults, but feel free to leave them if you really want to, for emphasis.

Reply:
I like to make it explicit what the inputs are, just for good measure.